### PR TITLE
Cleaned up Spanish translation of Java do-while guide

### PR DESCRIPTION
**Pull request by lyurrutia**
- [ x ] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [ x ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ x ] My pull request targets the `master` branch of freeCodeCamp.
- [ x ] None of my changes are plagiarized from another source without proper attribution.
- [ x ] My article does not contain shortened URLs or affiliate links.

### CODE REVIEW

File: guide/spanish/java/loops/do-while-loop/index.md [hunk 1]
```diff
@@ -35,13 +35,13 @@ Salida:
     iter_DoWhile Value: 21 
 ```
 
-**Recuerde** : la condición de un bucle `do-while` while se comprueba después de que el cuerpo del código se ejecuta una vez.
+**Recuerde** : la condición de un bucle `do-while` se comprueba después de que el cuerpo del código se ejecutado una vez.
```
Comment (hestellez) [inline guide/spanish/java/loops/do-while-loop/index.md]: Más bien **es ejecutado**

File: guide/spanish/java/loops/do-while-loop/index.md [hunk 2]
```diff
@@ -35,13 +35,13 @@ Salida:
...
 
 ![:rocket:](//forum.freecodecamp.com/images/emoji/emoji_one/rocket.png?v=2 ":cohete:") [Ejecutar código](https://repl.it/CJYl/0)
 
 ## Ejercicio
 
-¿Puedes adivinar la salida del siguiente fragmento de código?
+¿Puedes adivinar el resultado del siguiente fragmento de código?
```
Comment (hestellez) [inline guide/spanish/java/loops/do-while-loop/index.md]: Salida era ya una traducción aceptable, no?

**Comment by RandellDawson** [comment]
Thank you for  creating this PR.  In addition to needing to fix the issues pointed out by @hestellez, since this PR was created, the underlying document has changed.  Can you please review and resolve the current merge conflicts?  It could be the changes to the underlying document are so significant that your changes are no longer relevant.  If the latter is the case, it might be easier to just close this PR and create a new PR with any changes you still feel are important to make.

If you need any assistance or have questions about resolving the merge conflicts, please reply back or contact someone in the [Contributors Room on Gitter](https://gitter.im/FreeCodeCamp/Contributors).

Thank you.

**Comment by QuincyLarson** [comment]
We are restructuring a lot of the code and learning resources referenced in your pull request, so for now I am going to go ahead and close this pull request. Thanks again for opening this pull request and for contributing to the community.

**Pull request closed without merging**